Makefile

```diff
@@ -0,0 +1,58 @@
+.DEFAULT_GOAL := check
+
+# tools
+curl ?= curl
+xpath ?= xpath
+cddl ?= cddl
+diag2cbor ?= diag2cbor.rb
+
+# upstream repo configuration
+eat_repo_baseurl ?= https://ietf-rats-wg.github.io/eat
+# Set eat_repo_branch if you need to work with a branch other than master/main.
+# For example, if you need to work with a branch named CoSWID, do:
+#   make eat_repo_branch=CoSWID
+# Check https://ietf-rats-wg.github.io/eat/ for the branches that are currently
+# available.
+eat_repo_branch ?=
+ifeq ($(eat_repo_branch),)
+  eat_repo := $(eat_repo_baseurl)
+else
+  eat_repo := $(eat_repo_baseurl)/$(eat_repo_branch)
+endif
+
+eat_xml := draft-ietf-rats-eat.xml
+CLEANFILES += $(eat_xml)
+
+eat_cddl := eat.cddl
+CLEANFILES += $(eat_cddl)
+
+$(eat_cddl): $(eat_xml)
+	$(xpath) -n -q -e '//section[@anchor="collected-cddl"]//artwork/text()' $< \
```

Comment:
xpath on MacOS cares about the order of the arguments so the $< must be before the '-n -q ...' or you get an error that says "file -n not found"

Reply:
mmm I'm on Mac:
```
$ uname -a
Darwin tf 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64
```
and have the following `xpath(1)` installed: 
```
$ xpath
Usage:
/usr/bin/xpath5.28 [options] -e query [-e query...] [filename...]

If no filenames are given, supply XML on STDIN. You must provide at
least one query. Each supplementary query is done in order, the
previous query giving the context of the next one.

Options:

-q quiet, only output the resulting PATH.
-s suffix, use suffix instead of linefeed.
-p postfix, use prefix instead of nothing.
-n Don't use an external DTD.
```